src/components/infoMenu/index.jsx

```diff
@@ -52,12 +53,7 @@ class InfoMenu extends React.Component {
       case "About":
         return <About onOK={this.handleCloseModal} />;
       case "Help":
-        window.open(
-          "https://epad.stanford.edu/documentation/user",
```

Comment:
we talked about moving this to somewhere. I don't see it anywhere in the modified code